public/scripts/controllers/oneBookController.js

```diff
@@ -0,0 +1,44 @@
+angular
+  .module("booksApp")
+  .controller("OneBookController",OneBookController);
+
+OneBookController.$inject = ['$http', '$routeParams', '$location'];
+function OneBookController(   $http,   $routeParams,   $location) {
+  var vm = this;
+  vm.book = {};
+  vm.editBook = {
+    title: "",
+    image: "",
+    author: "",
+    releaseDate: "",
+  }
+  $http({
+    method: "GET",
+    url: "https://super-crud.herokuapp.com/books/" + $routeParams.id,
+  }).then(function Success(response){
+    vm.book = response.data;
+  })
+  vm.makeCopy = function(book){
```

Comment:
It even looks like you're trying to call this from your frontend as makeEdit rather than makeCopy :(